### ISSUE TITLE
Extra elements in NumberBox's UIA tree

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#1853]
**Describe the bug**

**Steps to reproduce the bug**

1. Turn on Narrator. Keep narrator in items mode.
2. Launch XAML Controls Gallery.
3. Activate NumberBox from home page.
4. Navigate to each text box present and observe the Narrator's behavior.

**Expected behavior**

Narrator should read the items description on reaching the UI's.

**Actual behavior**
When we reach the UI element, Narrator remain silent for first time.
On further navigation, narrator describes the item of the pane.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72565828-8a390100-3867-11ea-99d9-3702e2c4913b.png)

**Version Info**

Xaml Controls Gallery version 1.2.12.0

NuGet package version: 

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/24650137)

**Comment by StephenLPeters** [comment]
@Teap FYI

**Comment by chingucoding** [comment]
It seems like that the NumberBoxAutomationPeer is causing issues here. As we present the NumberBox as `Spinner` and inside that, we have the textbox.

So what would the best way to fix this issue? If we are not a Spinner in the UIA tree, the increase/decrease buttons are problematic.

As to why it does not announce the first time: No idea. The Accessibility Insights app picks the name up properly, so I am inclined to say that there might be an issue with narrator. 

@ranjeshj @YuliKl @teaP Any ideas?

**Comment by chingucoding** [comment]
I'm very certain that this is because of narrator. When changing the AutomationControlType from `Spinner` to `Edit` narrator starts to announce the name of the complete NumberBox. @YuliKl Is that behavior intended?

Also given that users can also enter mathematical expressions such as "2+4*3", would the AutomationControlType `Edit` make more sense for NumberBox? Or should it at least depend on how the control is configured, e.g. are the up/down buttons visible or does it accept expressions?

**Comment by ranjeshj** [comment]
Tracked via[ internal bug ](https://microsoft.visualstudio.com/OS/_workitems/edit/24650137)on narrator

**Comment by evelynwu-msft** [comment]
This appears to be fixed (probably by https://github.com/microsoft/microsoft-ui-xaml/pull/3775).
